### PR TITLE
Feature/css updates publisher

### PR DESCRIPTION
**Pull request by roderickmorales**
Att:
@bderusha @waheedahmed @awais786 @tasawernawaz and @jareerahsan 

This PR is the first step towards SASS refactoring for easier maintenance and updates. I did not add/edit/delete any styles, so this PR needs a smoke test to assure that everything works as expected.

Thanks,
Roderick

### CODE REVIEW

File: course_discovery/static/sass/publisher/tables.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    .table-row {
+
+      &:nth-child(odd) {
+        .table-col {
+          background: #f2f2f2;
+        }
+      }
+    }
+  }
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/tables.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/tabs.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        }
+      }
+    }
+
+    
+
+    &.is-active {
+      display: block;
+    }
+  }
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/tabs.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/typography.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.hd-3 {
+  color: #646262;
+  font-size: 24px;
+  font-weight: 400;
+  margin-bottom: 10px;
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/typography.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/navigations.scss [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+      &.selected {
+        border-bottom: 4px solid #0ea6ec;
+        color: black;
+      }
+    }
+  }
+}
+// ------------------------
+// ADMINISTRATION NAV (END)
+// ------------------------
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/navigations.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/lib.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    @include border-left(1px solid);
+}
+
+.border-top {
+    border-top: 1px solid;
+}
+
+.actions {
+  margin-top: 30px;
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/lib.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/forms.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    li {
+      display: inline-block;
+      @include margin-right(10px);
+
+      &:last-child {
+        @include margin-right(0);
+      }
+    }
+  }
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/forms.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/color-palette.scss [hunk 7]
```diff
@@ -0,0 +1 @@
+$light-gray: rgba(204, 204, 204, 1);
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/color-palette.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/buttons-links.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    border-radius: 5px;
+}
+
+.btn-courserun-add {
+    @include padding(3px, 16px, 3px, 16px);
+    @include margin-left(10px);
+    background-color: #169bd5;
+    border-color: #169bd5;
+    border-radius: 5px;
+}
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/buttons-links.scss]: Add new line at the end of file.

File: course_discovery/static/sass/publisher/buttons-links.scss [hunk 9]
```diff
@@ -0,0 +1,45 @@
+.course-breadcrumb{
+  padding-bottom: 20px;
+  a {
+    text-decoration: underline;
+  }
+}
+
+.btn-cancel {
+  padding-right: 10px;
+  padding-left: 10px;
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/buttons-links.scss]: Add RTL support on both.

**Comment by awais786** [comment]
@roderickmorales radio buttons are appearing vertically now. It new course page.

screen shot 2017-02-02 at 4 35 18 pm

**Comment by awais786** [comment]
Same thing happen for COURSE PACING type buttons.

**Comment by awais786** [comment]
screen shot 2017-02-02 at 7 32 16 pm

**Comment by roderickmorales** [comment]
@awais786 @waheedahmed I implemented your feedback.

**Review by awais786 — APPROVED** [review]

File: course_discovery/static/sass/publisher/tabs.scss [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/tabs.scss:144]: Add new line at the end of file here also.

**Review by waheedahmed — APPROVED** [review]
One minor comment otherwise 👍

**Pull request merged**